lib/Doctrine/DBAL/Types/FloatType.php

```diff
@@ -29,4 +30,12 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
     {
         return $value === null ? null : (float) $value;
     }
+
+    /**
+     * {@inheritdoc}
```

Comment:
Is this necessary?

Reply:
@morozov 

Just following convention set in IntegerType - do you want me to remove the inheritdoc?:

[https://github.com/doctrine/dbal/blob/9ff47e738575ab208f3ab4031412c58eb676ed1e/lib/Doctrine/DBAL/Types/IntegerType.php#L38](https://github.com/doctrine/dbal/blob/9ff47e738575ab208f3ab4031412c58eb676ed1e/lib/Doctrine/DBAL/Types/IntegerType.php#L38)